package.json

```diff
@@ -2,7 +2,7 @@
   "name": "rancher-desktop",
   "version": "0.0.1",
   "scripts": {
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint --no-fix",
```

Comment:
This seems to be the wrong PR for it.

Reply:
This will become obsolete anyway with eslint, but I was thinking that when we do lint in CI it shouldn't correct style issues but complain about them and force the developer to fix them.